deltachat-ffi/src/lib.rs

```diff
@@ -40,7 +40,7 @@ use deltachat_jsonrpc::yerpc::{OutReceiver, RpcClient, RpcSession};
 use message::Viewtype;
 use num_traits::{FromPrimitive, ToPrimitive};
 use tokio::runtime::Runtime;
-use tokio::sync::RwLock;
+use tokio::sync::{Mutex, RwLock};
```

Comment:
You don't need tokio mutex, just a `std::sync::Mutex` (or `parking_lot::Mutex` to avoid unwrap(), but then you need to add it as a dependency for deltachat_ffi). You call `.blocking_lock()` anyway.

See also https://docs.rs/tokio/latest/tokio/sync/struct.Mutex.html#which-kind-of-mutex-should-you-use, even in async code you don't necessarily need an async mutex.